src/ClapCommandLineHandlerTests.package/ClapCommandLineHandlerTest.class/instance/testCommandNameError.st

```diff
@@ -0,0 +1,5 @@
+tests
+testCommandNameError
+	| arguments |
+	arguments:= #('Evala').
+	self assert: ((self classToTest ) startHandler: arguments) equals: Error
```

Comment:
There is `#assert: aBlock raises: anException` (or something close) for testing exceptions.